PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,3 @@
-- #TICKET_NUMBER
 
 ### Reviewers
```

Comment:
I don't think we use the functinoal vs readability reviewer designations consistently.  I'd be okay with eliminating this section entirely and just relying upon Github's reviewers list.  If a particular reviewer is needed for a particular _type_ of review, we can just note that in the Notes for Reviewer section, perhaps.

Reply:
sounds good to me i can remove this